integration/aggregate_documents_compat_test.go

```diff
@@ -1618,20 +1618,20 @@ func TestAggregateCompatProject(t *testing.T) {
 		"TypeRecursive": {
 			pipeline: bson.A{
 				bson.D{{"$sort", bson.D{{"_id", -1}}}},
-				bson.D{{"$project", bson.D{{"type", bson.D{{"$type", bson.D{{"$type", bson.D{{"$type", "$v"}}}}}}}}}},
```

Comment:
Does this not work? Or is this not a meaningful test case? From my understand 2 level of nesting and 3 level of nesting just gives the same response :hugs:

Reply:
It worked before and it works currently, the issue here is that my goal here was to check for recursion and not for double recursion. Also 2level of nesting returns the same results as 3level